suite-native/biometrics/src/biometricsAtoms.ts

```diff
@@ -7,6 +7,8 @@ const isBiometricsOptionEnabledAtom = atomWithUnecryptedStorage<boolean>(
     false,
 );
 
+const hasDecidedBiometricsAtom = atomWithUnecryptedStorage<boolean>('hasDecidedBiometrics', false);
```

Comment:
nit: what do you think about `isBiometricsInitialSetupFinished` - is better IMO but if you don't like it, feel free to resolve here

Reply:
changed in 3f2db7d